tests/integration/README.md

```diff
@@ -32,41 +34,40 @@ systemctl stop firewalld
 systemctl disable firewalld
 ```
 
-- Install Python and its dependencies
-
-```shell script
-# Install dependencies
-yum install make gcc policycoreutils-python automake autoconf libtool epel-release git which sudo wget -y
-
-# Install development dependencies for jq library
-yum groupinstall "Development Tools" -y
+- Install Python3
 
-# Install Python3
+```sheel script
 yum install python36 python36-pip python36-devel -y
-
-# Install Python libraries
-pip3 install pytest freezegun jq jsonschema pyyaml==5.3 psutil distro pandas==0.25.3 pytest-html==2.0.1 numpydoc==0.9.2
 ```
 
-- Add some internal options and restart
```

Comment:
Apart from installing dependencies,
Shouldn't the setting of internal options be retained?

Reply:
Done 15215b391ba5d18eae8923b418137fd9f649b541